go-controller/pkg/ovn/management-port.go

```diff
@@ -184,6 +185,16 @@ func configureManagementPort(clusterSubnet []string, clusterServicesSubnet,
 		}
 	}
 
+	// Add a neighbour entry on the K8s node to map routerIP with routerMAC. This is
+	// required because in certain cases ARP requests from the K8s Node to the routerIP
+	// arrives on OVN Logical Router pipeline with ARP source protocol address set to
+	// K8s Node IP. OVN Logical Router pipeline drops such packets since it expects
+	// source protocol address to be in the Logical Switch's subnet.
+	_, _, err = util.RunIP("neigh", "add", routerIP, "dev", interfaceName, "lladdr", routerMac)
```

Comment:
I suppose,  you seen that this does not expire?

Reply:
@alinbalutoiu @aserdean I hope this commit does not break windows?

Reply:
> I suppose, you seen that this does not expire?

That is correct. Since they are permanent entries they never expire. When the OVN management port is removed, the neigh entries are removed as well.

Reply:
> @alinbalutoiu @aserdean I hope this commit does not break windows?

AFAICT it shouldn't break windows since the above code will not be called in Windows environment. If Windows chooses the right Source Protocol Address in ARP requests, the we might not need the equivalent of this windows at all. Just the static routes to OVN Distributed Router should suffice.

Reply:
Right. The question is does Windows chooses to use the right SPA in ARP request. If it behaves the same way as Linux, we will need a corresponding entry to Windows too.